src/components/accordion-item/accordion-item.tsx

```diff
@@ -86,8 +86,23 @@ export class AccordionItem implements ConditionalSlotComponent {
   /** Specifies a description for the component. */
   @Prop() description: string;
 
-  /** Specifies an icon to display - accepts Calcite UI icon names. */
-  @Prop({ reflect: true }) icon?: string;
+  /**
+   * Specifies an icon to display - accepts Calcite UI icon names.
+   *
+   * @deprecated use `iconStart` or `iconEnd` properties instead
+   */
+  @Prop({ mutable: true, reflect: true }) icon?: string;
+
+  @Watch("icon")
+  iconHandler(value: string): void {
+    this.iconStart = value;
+  }
+
+  /**Specified an icon to display at the start - accepts Calcite UI icon names. */
+  @Prop({ reflect: true }) iconStart?: string;
+
+  /**Specified an icon to display at the end - accepts Calcite UI icon names. */
+  @Prop({ reflect: true }) iconEnd?: string;
```

Comment:
Should updating iconEnd update icon/iconPosition? Same with iconstart. I see that its updated on connectedCallback but not anytime after.

Reply:
Either ways we have to make changes to the current implementation.